app/handlers/ErrorHandlerWithReturnLinkToManage.scala

```diff
@@ -17,25 +17,30 @@
 package handlers
 
 import config.FrontendAppConfig
+
 import javax.inject.{Inject, Singleton}
 import models.Link
 import play.api.i18n.{I18nSupport, Lang, MessagesApi}
-import play.api.mvc.Request
+import play.api.mvc.{Request, RequestHeader}
 import play.twirl.api.Html
 import views.html.{error_template, error_template_page_not_found}
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Introduce (implicit ec: ExecutionContext) as an injectable class inside of class constructor instead.